src/test/java/org/fundacionjala/sfdc/stepdefinition/AccountStepsCheck.java

```diff
@@ -84,9 +84,9 @@ public void iChooseAnAccount() {
     /**
      * iChooseAnAccountAndPressTheEditButton.
      */
-    @When("^I choose an account and press the edit button$")
+    @When("^I click on edit button to new account created$")
     public void iChooseAnAccountAndPressTheEditButton() {
-        mainPage.getAccountNameLink();
+      //  mainPage.getAccountNameLink();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)